hermes-api/src/main/java/pl/allegro/tech/hermes/api/SubscriptionMetrics.java

```diff
@@ -51,7 +79,7 @@ public void setState(Subscription.State state) {
 
     @Override
     public int hashCode() {
```

Comment:
hmm.. why do we need equals/hashcode on this object at all? i don't think the identity is ever relevant to us, each object is probably different by design. It would save us a lot of code